integrations/gcp/pyproject.toml

```diff
@@ -0,0 +1,105 @@
+[tool.poetry]
+name = "gcp"
+version = "0.1.0"
+description = "A GCP ocean integration"
+authors = ["Matan Geva <matang@getport.io>"]
+
+[tool.poetry.dependencies]
+python = "^3.11"
+port_ocean = { version = "^0.5.11", extras = ["cli"] }
+google-cloud-asset = "^3.25.1"
+ruff = "^0.3.4"
```

Comment:
?

Reply:
I've been getting weird errors localy and forgot to remove this once i pushed, did so now.